src/stylesheets/search/_browse.scss

```diff
@@ -0,0 +1,131 @@
+.browse {
+  padding: 1.5rem 2rem;
+  background: white;
+}
+
+.browse-heading {
+  margin: 0;
+  padding: 0;
+  font-size: 1.25rem;
+  margin-bottom: 1rem;
+}
+
+.browse-list {
+  margin: 0;
+  padding: 0;
+}
+
+.browse-item {
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+}
+
+.browse-button {
+  width: 2.5rem;
+  height: 2.5rem;
+  text-align: center;
+  display: flex;
+  justify-content: space-around;
+  line-height: 1;
+  background: none;
+  border: none;
+  color: $main-3;
+  font-size: 1.15rem;
+  font-weight: 600;
+
+  &:hover {
+    background: $main-5;
+    border: solid 1px $main-4;
+    border-radius: 4px;
+
+    cursor: pointer;
+  }
+}
+
+.browse-filter-link {
```

Comment:
Looks like an empty class.  Could be removed.

Reply:
+1 removed.